_Frontmatter/foreword.md

```diff
@@ -4,9 +4,10 @@ title: Foreword
 navOrder: 2
```

Comment:
```suggestion
navOrder: 1
```

Reply:
This variable also controls the menu order in the sidebar, and right now the "abstract" is set as the first landing page:

https://pages.nist.gov/FIPS201/abstract/

With a bit of tweaking we can have them be in different orders, but this was the agreed upon order for the menus from before.